src/assets/locales/en/translation.json

```diff
@@ -45,6 +45,13 @@
     "Minimum DAO bounty": "Minimum DAO bounty:",
     "Amount": "Amount:",
     "Amount Redeemed": "Amount Redeemed:",
+    "No Results": "No Results",
+    "Search": "Search",
+    "Validate Address" : "Please enter a valid address",
+    "Validate HEX" : "Must be a hex value",
+    "Validate Digits" : "Must contain only digits",
+    "Call to Contract": "This proposal will call the following contract:",
+    "No ABI": "No ABI for this contract",
```

Comment:
mainnet/rinkeby/kovan : "No ABI found for target contract 0x...  please verify contract in etherscan.io"
xdai : "No ABI found for target contract 0x...  please verify contract in blocksscount.com"